libsolidity/lsp/Transport.cpp

```diff
@@ -58,6 +59,12 @@ optional<Json::Value> Transport::receive()
 
 	string const data = readBytes(stoul(headers->at("content-length")));
 
+	if (m_traceLogFilePath)
+	{
+		ofstream traceLogger(m_traceLogFilePath.value().generic_string(), ios::app);
```

Comment:
Hmm.. just wondering.. if `generic_string` is basically not the native path format - isn't it possible that a path returned here is not compatible with `ofstream`? so that `.string()` should be used here - `returns the path in native pathname format converted to a string`. Not sure though. @cameel?